network-api/app/networkapi/news/models.py

```diff
@@ -57,8 +57,8 @@ class News(models.Model):
         help_text='URL link to the article/media clip',
     )
     excerpt = models.TextField(
-        max_length=1000,
-        help_text='A small extract from the article',
+        max_length=200,
```

Comment:
since this is a `TextField`, no max length is enforced at the db level...the `200` will only be enforced at the model level so it's relatively safe to change.

Reply:
Limit per design is 146.

Reply:
See comment in original thread. 200 is approved. We all know how being super strict about web content length is unrealistic with regards to layout, so there's some flex space here.

Reply:
K